### PR TITLE
Move Edge runs to Azure pipelines

### PR DESCRIPTION
**Pull request by mustjab**
Moving Edge WPT.fyi runs from custom built pipeline to Azure pipelines that are used by other runs. We no longer need to maintain a custom pipeline since Azure pipelines now supports OS images and features required for us to do Edge runs. Validated this change with test runs for Edge Stable, Dev and Canary channels.

Stable: https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=80546&view=results
Dev: https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=80554&view=results
Canary: https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=80575&view=results

### CODE REVIEW

**Comment by mustjab** [comment]
Discussion of this change can be found here: https://github.com/web-platform-tests/rfcs/pull/102

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -401,7 +374,7 @@ jobs:
     displayName: 'Publish results'
     inputs:
       artifactName: 'edge-stable-results'
-  - template: tools/ci/azure/cleanup_win10.yml
```
Comment (foolip) [inline .azure-pipelines.yml:404]: Can you remove this file? I presume it's unused now.

File: .azure-pipelines.yml [hunk 2]
```diff
@@ -381,11 +352,13 @@ jobs:
...
   pool:
-    name: 'Hosted Windows Client'
+    vmImage: 'windows-2019'
   steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.9'
   - template: tools/ci/azure/system_info.yml
   - template: tools/ci/azure/checkout.yml
-  - template: tools/ci/azure/install_python.yml
```
Comment (foolip) [inline .azure-pipelines.yml:388]: Can you remove this file?

File: .azure-pipelines.yml (hunk 1 above)
Comment (mustjab) [reply .azure-pipelines.yml:404]: Yes, removed

File: .azure-pipelines.yml (hunk 2 above)
Comment (mustjab) [reply .azure-pipelines.yml:388]: Removed

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@DanielRyanSmith you'll get some conflicts from this change, and can you also make sure the newly added instances of 3.9 are updated?

**Comment by foolip** [comment]
https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=80654&view=results is the first scheduled run after this...

**Comment by foolip** [comment]
Looking at that, I wonder if we're going to run into a problem with parallelism, and PR jobs being in a queue behind hour-long jobs at times.

@mustjab a long time ago we got the parallelism raised via an internal request, can you check what the current limit is? It looks like we have 20 jobs running in parallel now, but that's not enough for the jobs triggered by `epochs/three_hourly` and will be even further from the required parallelism when weekly, daily and 3-hourly runs are triggered at the same time once a week.

**Comment by foolip** [comment]
And to spell it out, I assume that limit does not apply to the custom agent pool, so previously it had to be enough for Safari runs only, which should be at most 14 jobs in parallel.

**Comment by mustjab** [comment]
This is the limit that I see documented: https://docs.microsoft.com/en-us/azure/devops/pipelines/licensing/concurrent-jobs?view=azure-devops&tabs=ms-hosted 

We can just stick with either Dev or Canary channels for now to reduce the resources used by Edge jobs.

**Comment by foolip** [comment]
OK, we must have gotten the limit raised from 10 to 20 back then, via an internal request to the Azure Pipelines team.

At minimum we'll have some definition of stable+experimental for both Edge and Safari, which means 4 runs at the same time. We will definitely exceed the limit of 20 at times, but maybe we should try to quantify how much it delays PRs?

**Pull request merged**